modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -338,10 +338,52 @@
             OpenStruct.new(
               id: '0958d973-36fb-43ef-8801-2718bd33c825',
               evss_id: '111111111',
-              status: 'pending'
+              status: 'pending',
+              veteranId: '1013062086V794840'
             )
           end
 
+          context 'and is not associated with the current user' do
```

Comment:
These tests should really probably function more like this because we're now depending on some ActiveRecord logic to help us validate that a claim belongs to the Veteran.  So we should really be putting records into the DB and then making sure we can/can't find them as necessary.
```
          context 'and is not associated with the current user' do
            let(:not_vet_lh_claim) do
              create(:auto_established_claim, veteran_icn: '123')
            end
            let(:mismatched_claim_veteran_path) do
              "/services/claims/v2/veterans/#{veteran_id}/claims/#{not_vet_lh_claim.id}"
            end

            it 'returns a 404' do
              with_okta_user(scopes) do |auth_header|
                get mismatched_claim_veteran_path, headers: auth_header

                expect(response.status).to eq(404)
              end
            end
          end

          context 'and is associated with the current user' do
            let(:bgs_claim) { nil }
            let(:lighthouse_claim) do
              create(:auto_established_claim, veteran_icn: veteran_id)
            end
            let(:matched_claim_veteran_path) do
              "/services/claims/v2/veterans/#{veteran_id}/claims/#{lighthouse_claim.id}"
            end

            it 'returns a 200' do
              with_okta_user(scopes) do |auth_header|
                VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
                  expect(ClaimsApi::AutoEstablishedClaim)
                    .to receive(:get_by_id_and_icn).and_return(lighthouse_claim)

                  get matched_claim_veteran_path, headers: auth_header

                  expect(response.status).to eq(200)
                end
              end
            end
          end
```